pandas_datareader/_testing.py

```diff
@@ -2,7 +2,7 @@
 Utilities for testing purposes.
```

Comment:
don't change the mode of the file, use

``git update-index --chmod -x <file>`` to put it back